frame/support/procedural/src/storage_alias.rs

```diff
@@ -549,6 +595,33 @@ fn generate_storage_instance(
 	let name = Ident::new(&format!("{}_Storage_Instance", storage_name), Span::call_site());
 	let storage_name_str = storage_name.to_string();
 
+	let mut counter_code = Option::None;
+
+	if is_counted_map {
+		let counter_name = Ident::new(&format!("Counter_{}", name), Span::call_site());
+		let counter_storage_name_str = "counter_".to_owned() + &storage_name_str;
```

Comment:
Could you please use this function there https://github.com/paritytech/substrate/blob/4fadc98e1fa6b3cbb158735007d46bfe48a74158/frame/support/procedural/src/pallet/expand/storage.rs#L44

Move it somewhere in the crate to reuse it

Reply:
good spot, going to check this out!